src/js/modules/rpc/server.ts

```diff
@@ -98,6 +99,10 @@ export class ProcessBatchServer extends SupervisorServer {
     return Promise.resolve({ responses });
   }
 
+  heartbeat(input: EmptyRequest): Promise<EmptyResponse> {
+    return Promise.resolve(new EmptyResponse());
```

Comment:
it should be,   ` return Promise.resolve({empty: 0})`, the `new EmptyResponse()` doesn't create a empty value, and it needs for encoding functions on rpc layer